app/assets/stylesheets/govuk_publishing_components/components/govspeak/_typography.scss

```diff
@@ -56,25 +56,25 @@
   }
 
   &.direction-rtl blockquote {
-    padding: 0 $gutter-two-thirds 0 0;
+    padding: 0 govuk-spacing(4) 0 0;
 
     @include govuk-media-query($from: desktop) {
-      margin: 0 (-$gutter) 0 0;
+      margin: 0 (- govuk-spacing(6)) 0 0;
```

Comment:
```suggestion
      margin: 0 -(govuk-spacing(6)) 0 0;
```

Reply:
Added all the suggestions except for this one which needs to be `margin: 0 (- govuk-spacing(6)) 0 0` to pass both linters, otherwise it will try to subtract it from 0.